x-pack/plugins/security_solution/server/ui_settings.ts

```diff
@@ -147,6 +149,7 @@ export const initUiSettings = (uiSettings: CoreSetup['uiSettings']) => {
         defaultMessage: 'News feed URL',
       }),
       value: NEWS_FEED_URL_SETTING_DEFAULT,
+      sensitive: true,
```

Comment:
I believe that the setting for IP reputation URIs (`IP_REPUTATION_LINKS) could also contain sensitive information in the same way as the newsfeed URL.